### PR TITLE
Fix the logic of when ToggleListingCommitsAction gets disabled

### PR DESCRIPTION
**Pull request by PawelLipski**

### CODE REVIEW

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/ToggleListingCommitsAction.java [hunk 1]
```diff
@@ -45,16 +44,6 @@ public void onUpdate(AnActionEvent anActionEvent) {
       return;
     }
 
-    boolean noChildBranchExists = branchLayout.get().getRootEntries()
-        .exists(rootBranch -> rootBranch.getChildren().isEmpty());
```
Comment (PawelLipski) [inline frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/ToggleListingCommitsAction.java:49]: There was a bug here... it should've been `.forAll`, not `.exists`

Comment (PawelLipski) [reply frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/ToggleListingCommitsAction.java:49]: But anyway, we don't need this check, `anyCommitExists == false` covers the case of no child branch as well

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/ToggleListingCommitsAction.java [hunk 2]
```diff
@@ -45,16 +44,6 @@ public void onUpdate(AnActionEvent anActionEvent) {
       return;
```
Comment (micpiotrowski) [inline frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/ToggleListingCommitsAction.java:44]: whoops... according to PR #515 on line 40 should be getBranchLayoutWith**out**Logging

**Review by micpiotrowski — CHANGES_REQUESTED** [review]

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/ToggleListingCommitsAction.java (hunk 2 above)
Comment (PawelLipski) [reply frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/ToggleListingCommitsAction.java:44]: Fixed... but it's super-hard to enforce, lemme solve that `systemically` on another PR :thinking:

Comment (PawelLipski) [reply frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/ToggleListingCommitsAction.java:44]: Ok, done on #534

**Review by mkondratek — APPROVED** [review]

**Review by micpiotrowski — APPROVED** [review]

**Pull request merged**